fortanix-vme/eif-tools/README.md

```diff
@@ -0,0 +1,19 @@
+# EIF utilities
+
+Compiles with Rust nightly.
+
+## ftxvme-elf2eif
+
+`ftxvme-elf2eif` converts ELF executables to AWS Nitro's EIF format.
+
+## eif-sign
```

Comment:
Do we want to keep this name or change it to `ftxvme-eifsign` or something?

Reply:
Probably you were thinking of being consistent with `ftxvme-elf2eif`, but I think the difference is that `ftxvme-elf2eif` is tailored to ELF files produced by our `fortanixvme` rust target (the tool might work for other ELF files as well, but we only claim compatibility with ELF files produced by our own rust target) while `eif-sign` should work for any EIF.

For SGX we also have `sgxs-sign`, and `ftxsgx-elf2sgxs`.